app/assets/stylesheets/application.css.scss

```diff
@@ -26,7 +26,9 @@
   }
 
   li {
-    background-color: #f00;
+    background-color: #faebd7;
+    border: .25em solid #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.25` should be written with a leading zero as `0.25`